aws-amplify/amplify-js
# feat(@ui-components/username-attributes): Add usernameAlias support

**_Description of changes:_** Adds usernameAlias support.

**_How to test:_**
- run `yarn start` on `packages/ui-components`
- make sure to use `aws-exports` with auth setup.

**_Usage:_** `usernameAlias` can be one of these `email | phone_number | username`
`<amplify-authenticator usernameAlias="username" />`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
